lib/git.rb

```diff
@@ -57,7 +57,7 @@ def log(path = nil, revstring = Constants::HEAD, options = {})
       if path && options[:follow]
         cfg       = Configuration.new(nil)
         cfg.add_setting('renames', true, 'diffs', nil)
-        follow    = FollowFilter.create(path, cfg.get(org.eclipse.jgit.diff.DiffConfig::KEY))
+        follow    = FollowFilter.create(path, cfg.jconfig.get(org.eclipse.jgit.diff.DiffConfig::KEY))
```

Comment:
Can you explain this change? Did `cfg.get` work before or not at all?

Reply:
It worked when running the specs on RJGit, but not when trying to call this method from the rjgit-adapter. My guess is that context somehow breaks the method_missing magic that this relied on, so perhaps better to be explicit.

Reply:
Okay!